deploy/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM alpine:3.15 AS build
+FROM mirror.gcr.io/library/golang:1.18-alpine3.16 AS build
 
-RUN apk --no-cache add libc6-compat device-mapper findutils zfs build-base linux-headers python3 bash git wget cmake pkgconfig ndctl-dev && \
+# Install build depdencies for all supported arches
+RUN apk --no-cache add bash build-base cmake device-mapper findutils git \
```

Comment:
The dependencies should be identical, I just alphabetically

Reply:
thanks for cleanup!